adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -215,8 +215,7 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
 - (void)vungleSDKDidInitialize {
   VungleSDK *sdk = [VungleSDK sharedSDK];
   if ([VungleRouterConsent getConsentStatus] > 0) {
-    [sdk updateConsentStatus:[VungleRouterConsent getConsentStatus]
-       consentMessageVersion:[VungleRouterConsent getConsentMessageVersion]];
+    [sdk updateConsentStatus:[VungleRouterConsent getConsentStatus] consentMessageVersion:@""];
```

Comment:
Why are we sending an empty string as the consentMessageVersion? Is it not needed for the publisher to set this value?

Reply:
@maninomula Thanks for your feedback. Since Vungle uses `consentMessageVersion` for only native integration case, we decided to hide in mediation adapter level.